### PR TITLE
Hide zero balances in asset send flow

### PR DESCRIPTION
**Pull request by antonijzelinskij**
#85ztt1vta
#85ztt1wf3
image

### CODE REVIEW

**Comment by stepanLav** [comment]
Task linked: [CU-85ztt1vta Android. Don't show zero balances in send flow](https://app.clickup.com/t/85ztt1vta)

File: feature-assets/src/main/java/io/novafoundation/nova/feature_assets/domain/assets/search/AssetSearchInteractor.kt [hunk 1]
```diff
@@ -38,9 +39,11 @@ class AssetSearchInteractor(
         queryFlow: Flow<String>,
         externalBalancesFlow: Flow<List<ExternalBalance>>,
     ): Flow<Map<AssetGroup, List<AssetWithOffChainBalance>>> {
-        return searchAssetsInternalFlow(queryFlow, externalBalancesFlow) {
-            val chainAsset = it.token.configuration
-            assetSourceRegistry.sourceFor(chainAsset).transfers.areTransfersEnabled(chainAsset)
+        return searchAssetsInternalFlow(queryFlow, externalBalancesFlow) { asset ->
+            val chainAsset = asset.token.configuration
+            asset.freeInPlanks.isPositive() &&
```
Comment (ERussel) [inline feature-assets/src/main/java/io/novafoundation/nova/feature_assets/domain/assets/search/AssetSearchInteractor.kt:44]: User can send a token if it has positive transferable balance

File: feature-assets/src/main/java/io/novafoundation/nova/feature_assets/presentation/send/flow/AssetSendFlowViewModel.kt [hunk 2]
```diff
@@ -38,4 +43,15 @@ class AssetSendFlowViewModel(
         val assePayload = AssetPayload(chainAsset.chainId, chainAsset.id)
         router.openSend(assePayload)
     }
+
+    override fun getPlaceholder(query: String, assets: List<Any>): PlaceholderModel? {
+        if (query.isEmpty() && assets.isEmpty()) {
+            return PlaceholderModel(
+                text = resourceManager.getString(R.string.assets_flow_placeholder),
+                imageRes = R.drawable.ic_no_search_results
+            )
```
Comment (valentunn) [inline feature-assets/src/main/java/io/novafoundation/nova/feature_assets/presentation/send/flow/AssetSendFlowViewModel.kt:52]: Should there by a call to action (Buy)? Or you plan to do it in a separate PR?

Comment (antonijzelinskij) [reply feature-assets/src/main/java/io/novafoundation/nova/feature_assets/presentation/send/flow/AssetSendFlowViewModel.kt:52]: I'll do it here

**Review by ERussel — APPROVED** [review]
